pkg/proxy/service/cluster_service.go

```diff
@@ -42,7 +42,7 @@ func NewMemberClusterService(context servicecontext.ServiceContext, options ...O
 
 func (s *ServiceImpl) GetNamespace(ctx *gin.Context, userID string) (*namespace.NamespaceAccess, error) {
 	// Get Signup
-	signup, err := s.ServiceContext.Services().SignupService().GetSignup(userID)
+	signup, err := s.ServiceContext.Services().SignupService().GetSignup(userID, "")
```

Comment:
I just looked through this code and I think you're right.. I'll make the changes required for this, it will mean quite a bit of work to fix the tests here as they're geared towards using just the UserID.

Reply:
This is done now.